README.md

```diff
@@ -55,69 +48,37 @@ While RFCs are intended for "significant" changes, we recommend trying to keep e
-    * TODO: Should this be automated?
-1. A new "work-in-progress" (WIP) pull request will be created with that updates the RFC's status to `approved`
-    * TODO: This should probably be automated
-1. In the WIP pull request, your proposed RFC will now be up for feedback from the OpenTelemetry community. Others may suggest changes, raise concerns, or add new pull requests building on top of yours
-1. Aim to address the discussion! While you are the champion of the RFC, the community collectively owns the result. We're all in this together :)
-    * Please make changes as new commits, documenting the reasoning for each in the commit message. Try to avoid squashing, rebasing, force-pushing, etc. at this point, as doing so may mess with the historical discussion on GitHub
-1. Once the value and trade-offs of the change have been sufficiently discussed, a non-author member of the relevant SIG will remove "WIP" from the PR's title and propose a "motion for final comment period" (FCP)
-    * By this point, no **new** points should have been raised in the discussion for **at least three business days**
-    * If substantial new points (e.g., a significant trade-off) are raised *during* the FCP, the FCP should be cancelled and discussion renewed
-1. During the FCP, SIG members should each vote to `approve`, `reject`, or `defer` the RFC
-    * If the discussion has been contentious, members should explain the rationale behind their vote
-1. The FCP will be considered closed and the RFC either approved, rejected, or approved after the following conditions have all been met:
-    * At least two business days have passed since the FCP began
-    * Either:
-      * A majority of non-author members have voted
-      * At least three non-author members have voted, and the vote is unanimous
-1. The RFC will be merged with the appropriate state
-
-### Reviewing an RFC
-
-Discussion should primarily take place in the pull request aimed at approving the RFC. Any discussion that takes place outside of GitHub - e.g., in meetings or community chat - should be summarized and added to the pull request.
-
-### Implementing an RFC
-
-Once an RFC has been approved, a corresponding issue should be created and prioritized accordingly in the relevant repository.
+### Writing an RFC
 
-The original author is welcome but not obligated to implement it. If you are interested in implementing an RFC, please mention it in either the RFC pull request or the new issue (depending on which is active by then). While authors who are interested in implementation generally get priority, we encourage anyone who's interested to collaborate together!
+* First, [fork](https://help.github.com/en/articles/fork-a-repo) this [repo](https://github.com/open-telemetry/oteps).
+* Copy [`0000-template.md`](./0000-template.md) to `text/0000-my-rfc.md`, where `my-rfc` is a title relevant to your proposal. Leave the number as is for now.
```

Comment:
This never says when the number will change from 0000. I suggest implementing #32 as part of this PR, since they are closely related.

Reply:
Good catch. Changing the RFC ID to match the PR _is_ described below, under "Submitting an RFC," but it's worth it to mention here as well.